core/include/traccc/edm/impl/track_fit_collection.ipp

```diff
@@ -0,0 +1,21 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022-2025 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+namespace traccc::edm {
+
+template <typename BASE>
+TRACCC_HOST_DEVICE void track_fit<BASE>::reset_quality() {
+
+    ndf() = {};
+    chi2() = {};
+    pval() = {};
+    nholes() = {};
+}
```

Comment:
Please don't make an `.ipp` file for this; fold it into the `.hpp` file.

Reply:
Since I already made the file, it would be silly to remove it. Even if you don't like them, I myself do like `.ipp` files.

Reply:
lol